lib/active_model_serializers/adapter/json_api/deserialization.rb

```diff
@@ -1,12 +1,12 @@
+require 'jsonapi'
+
 module ActiveModelSerializers
   module Adapter
     class JsonApi
       # NOTE(Experimental):
       # This is an experimental feature. Both the interface and internals could be subject
       # to changes.
       module Deserialization
```

Comment:
At the end of this PR, this should be removed / deprecated

/cc @richmolj  - rumor has it you want to tackle moving deserialization out?